mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -26,24 +25,61 @@ fun PlayScreen(
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(),
 ) {
+  val buttonState = rememberExpandableFloatingActionButtonState()
+
   Box(modifier = modifier.fillMaxSize().padding(contentPadding)) {
-    NewGameButton(onNewGame = state.onNewGameClick, modifier = Modifier.align(Alignment.BottomEnd))
+    NewGameButton(
+        state = buttonState,
+        onLocalGame = { state.onLocalGameClick() },
+        onRemoteGame = { state.onOnlineGameClick() },
+        modifier = Modifier,
```

Comment:
You should preserve the alignment here and not have it in the button composable.

Reply:
Fixed in [0a0980e](https://github.com/epfl-SDP/android/pull/226/commits/0a0980ec39e8c6024b8d84dbfb41a6ce3a19de16)